src/components/Checkbox/Checkbox.st.css

```diff
@@ -27,6 +27,7 @@
     TextColor: --overridable;
     BorderColor: --overridable;
     IconColor: color-8;
+    BoxColor: color-1;
```

Comment:
I just noticed that you're not using `BoxColor` anywhere.
Why is that?

Reply:
You're right, I forgot about it, I'll add it.